roachtest: tpccbench/nodes=9/cpu=4/chaos/partition failed

[(roachtest).tpccbench/nodes=9/cpu=4/chaos/partition failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1704870&tab=buildLog) on [provisional_202001221958_v2.1.11@0f056a5fbfc2b8774f8567a97432900b13a408e6](https://github.com/cockroachdb/cockroach/commits/0f056a5fbfc2b8774f8567a97432900b13a408e6):

```
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2245
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.execCmd
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:399
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1855
		    main.(*cluster).RunE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1847
		    main.loadTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:628
		    main.runTPCCBench.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:718
		    main.(*monitor).Go.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: %s returned:
		    stderr:
		    %s
		    stdout:
		    %s
		    -- arg 1: <string>
		    -- arg 2: <string>
		    -- arg 3: <string>
		  - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1704870-1579731692-70-n10cpu4:10 -- ./workload fixtures load tpcc --warehouses=2000 --scatter --checks=false --partitions=3 {pgurl:1} returned:
		    stderr:
		    
		    stdout:
		    I200123 02:30:05.120541 1 ccl/workloadccl/cliccl/fixtures.go:279  starting restore of 9 tables
		    W200123 02:30:05.121295 1 ccl/workloadccl/fixture.go:461  error retrieving automatic stats cluster setting: pq: unknown setting: "sql.stats.automatic_collection.enabled"
		    I200123 02:30:06.448829 98 ccl/workloadccl/fixture.go:547  loaded 7.8 MiB table item in 1.327311171s (100000 rows, 0 index entries, 5.9 MiB)
		    I200123 02:30:32.835429 45 ccl/workloadccl/fixture.go:547  loaded 2.0 MiB table district in 27.713747729s (20000 rows, 0 index entries, 73 KiB)
		    I200123 02:31:07.090373 44 ccl/workloadccl/fixture.go:547  loaded 106 KiB table warehouse in 1m1.968763408s (2000 rows, 0 index entries, 1.7 KiB)
		    I200123 02:35:21.015791 49 ccl/workloadccl/fixture.go:547  loaded 254 MiB table new_order in 5m15.894335544s (18000000 rows, 0 index entries, 824 KiB)
		    I200123 02:42:21.723339 47 ccl/workloadccl/fixture.go:547  loaded 8.5 GiB table history in 12m16.601691723s (60000000 rows, 120000000 index entries, 12 MiB)
		    I200123 02:42:27.868254 48 ccl/workloadccl/fixture.go:547  loaded 2.7 GiB table order in 12m22.746479192s (60000000 rows, 60000000 index entries, 3.7 MiB)
		    I200123 02:54:21.344617 46 ccl/workloadccl/fixture.go:547  loaded 34 GiB table customer in 24m16.223217431s (60000000 rows, 60000000 index entries, 24 MiB)
		    I200123 02:55:32.433362 99 ccl/workloadccl/fixture.go:547  loaded 63 GiB table stock in 25m27.31175003s (200000000 rows, 200000000 index entries, 42 MiB)
		    I200123 03:15:12.820696 100 ccl/workloadccl/fixture.go:547  loaded 46 GiB table order_line in 45m7.699267988s (600008449 rows, 600008449 index entries, 17 MiB)
		    I200123 03:15:13.309148 1 ccl/workloadccl/cliccl/fixtures.go:286  restored 155 GiB bytes in 9 tables (took 45m8.188385315s, 58.48 MiB/s)
		    Error: Could not postload: could not partition tables: Couldn't exec "\n\t\t\tALTER INDEX item@replicated_idx_0\n\t\t\tCONFIGURE ZONE USING num_replicas = COPY FROM PARENT, constraints = '{\"+rack=0\":1}', lease_preferences = '[[+rack=0]]'": pq: syntax error at or near "from"
		    Error:  exit status 1:
		  - exit status 1

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/chaos/partition](https://teamcity.cockroachdb.com/viewLog.html?buildId=1704870&tab=artifacts#/tpccbench/nodes=9/cpu=4/chaos/partition)
Related:
- #41553 roachtest: tpccbench/nodes=9/cpu=4/chaos/partition failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fchaos%2Fpartition.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).tpccbench/nodes=9/cpu=4/chaos/partition failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1704888&tab=buildLog) on [provisional_202001221958_v2.1.11@0f056a5fbfc2b8774f8567a97432900b13a408e6](https://github.com/cockroachdb/cockroach/commits/0f056a5fbfc2b8774f8567a97432900b13a408e6):

```
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2245
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - error with attached stack trace:
		    main.execCmd
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:399
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1855
		    main.(*cluster).RunE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1847
		    main.loadTPCCBench
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:628
		    main.runTPCCBench.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tpcc.go:718
		    main.(*monitor).Go.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: %s returned:
		    stderr:
		    %s
		    stdout:
		    %s
		    -- arg 1: <string>
		    -- arg 2: <string>
		    -- arg 3: <string>
		  - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1704888-1579731699-70-n10cpu4:10 -- ./workload fixtures load tpcc --warehouses=2000 --scatter --checks=false --partitions=3 {pgurl:1} returned:
		    stderr:
		    
		    stdout:
		    I200123 02:40:50.911769 1 ccl/workloadccl/cliccl/fixtures.go:279  starting restore of 9 tables
		    W200123 02:40:50.912411 1 ccl/workloadccl/fixture.go:461  error retrieving automatic stats cluster setting: pq: unknown setting: "sql.stats.automatic_collection.enabled"
		    I200123 02:40:51.899694 31 ccl/workloadccl/fixture.go:547  loaded 2.0 MiB table district in 986.952543ms (20000 rows, 0 index entries, 2.0 MiB)
		    I200123 02:40:52.882150 84 ccl/workloadccl/fixture.go:547  loaded 7.8 MiB table item in 1.969269521s (100000 rows, 0 index entries, 3.9 MiB)
		    I200123 02:48:39.572519 30 ccl/workloadccl/fixture.go:547  loaded 106 KiB table warehouse in 7m48.659875132s (2000 rows, 0 index entries, 232 B)
		    I200123 02:48:46.776391 83 ccl/workloadccl/fixture.go:547  loaded 254 MiB table new_order in 7m55.863869592s (18000000 rows, 0 index entries, 547 KiB)
		    I200123 02:49:21.429619 82 ccl/workloadccl/fixture.go:547  loaded 2.7 GiB table order in 8m30.517007931s (60000000 rows, 60000000 index entries, 5.3 MiB)
		    I200123 02:56:56.903775 33 ccl/workloadccl/fixture.go:547  loaded 8.5 GiB table history in 16m5.991218355s (60000000 rows, 120000000 index entries, 9.1 MiB)
		    I200123 03:01:20.449879 32 ccl/workloadccl/fixture.go:547  loaded 34 GiB table customer in 20m29.537136411s (60000000 rows, 60000000 index entries, 29 MiB)
		    I200123 03:15:39.822775 86 ccl/workloadccl/fixture.go:547  loaded 46 GiB table order_line in 34m48.910250386s (600008449 rows, 600008449 index entries, 22 MiB)
		    I200123 03:19:37.425352 85 ccl/workloadccl/fixture.go:547  loaded 63 GiB table stock in 38m46.512462456s (200000000 rows, 200000000 index entries, 28 MiB)
		    I200123 03:19:37.955861 1 ccl/workloadccl/cliccl/fixtures.go:286  restored 155 GiB bytes in 9 tables (took 38m47.043857166s, 68.06 MiB/s)
		    Error: Could not postload: could not partition tables: Couldn't exec "\n\t\t\tALTER INDEX item@replicated_idx_0\n\t\t\tCONFIGURE ZONE USING num_replicas = COPY FROM PARENT, constraints = '{\"+rack=0\":1}', lease_preferences = '[[+rack=0]]'": pq: syntax error at or near "from"
		    Error:  exit status 1:
		  - exit status 1

```

<details><summary>More</summary><p>


Artifacts: [/tpccbench/nodes=9/cpu=4/chaos/partition](https://teamcity.cockroachdb.com/viewLog.html?buildId=1704888&tab=artifacts#/tpccbench/nodes=9/cpu=4/chaos/partition)
Related:
- #41553 roachtest: tpccbench/nodes=9/cpu=4/chaos/partition failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-master](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-master)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Atpccbench%2Fnodes%3D9%2Fcpu%3D4%2Fchaos%2Fpartition.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/tpcc.go
+++ b/pkg/cmd/roachtest/tpcc.go
@@ -376,6 +376,8 @@ func registerTPCC(r *testRegistry) {
 
 		LoadWarehouses: 2000,
 		EstimatedMax:   900,
+
+		MinVersion: "v19.1.0",
 	})
 }
 

